src/features/currentTodo.ts

```diff
@@ -1,10 +1,14 @@
-import { createSlice } from '@reduxjs/toolkit';
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
 import { Todo } from '../types/Todo';
 
 const initialState = null as Todo | null;
 
 export const currentTodoSlice = createSlice({
   name: 'currentTodo',
   initialState,
-  reducers: {},
+  reducers: {
+    setCurrentTodo: (_, { payload }: PayloadAction<Todo | null>) => payload,
```

Comment:
The `setCurrentTodo` reducer is correctly defined to update the state with the payload. Ensure that the `Todo` type includes all necessary properties like `id`, `title`, `completed`, and `userId`.